src/zip.ts

```diff
@@ -46,7 +46,9 @@ export async function* loadFiles(files: ForAwaitable<ZipFileDescription & Metada
   for await (const file of files) {
     yield fileHeader(file)
     yield file.encodedName
-    yield* fileData(file)
+    if (file.bytes) {
+      yield* fileData(file as ZipFileDescription & Metadata)
```

Comment:
That's weird. `file` can be two types, but the test should be enough for TypeScript to discriminate without our help.

Reply:
Yeah, it was needed before I did the hack with undefined bytes and crc. Not needed now.